identity/Cargo.toml

```diff
@@ -5,8 +5,13 @@ authors = ["Markus Merklinger <markus.merklinger@code.berlin>"]
 edition = "2018"
 
 [lib]
-name = "identity_rpc_service"
-path = "src/rpc/service.rs"
+name = "identity"
+path = "src/identity/lib.rs"
+
+[[bin]]
+name = "identity"
+test = false
```

Comment:
What is your reason to explicitly omit the tests? Especially since there are none?

Reply:
It's mostly a thing I got from the docs. The binary is supposed to use only library functions and have no own modules, which would require testing. Thus, I excluded it from checks. It doesn't really matter. Do you prefer to drop it, because of readability?